networkx/algorithms/tests/test_matching.py

```diff
@@ -400,6 +397,12 @@ def test_wrong_graph_type(self):
         raises(error, nx.max_weight_matching, nx.DiGraph())
         raises(error, nx.min_weight_matching, nx.DiGraph())
 
+    def test_min_weight_matching_max_cardinality(self):
+        G = nx.Graph()
+        G.add_weighted_edges_from([(1, 2, 1000), (2, 3, 2), (3, 4, 3000)])
```

Comment:
This test may need some additional context in order to understand why it's added, maybe something like:

```suggestion
        G.add_weighted_edges_from([(1, 2, 1000), (2, 3, 2), (3, 4, 3000)])
        # The minimum-weight maximal matching is {(2, 3)}; the minimum weight
        # maximum-cardinality matching is {(1, 2), (3, 4)}. See gh-8062.
```

Not really that important, but it might be nice to link to the discussion for anyone who stumbles across this in the future.